unit_tests/test_ouvrages.py

```diff
@@ -19,7 +19,7 @@ def test_one_document_pdf(self):
             OuvrageFile("document.pdf", "http://fake.url", datetime.date(2022, 9, 16)),
         )
 
-    def test_document_pdf_with_vignette_and_metadata(self):
+    def test_document_pdf_with_vignette_and_metadata_and_log(self):
```

Comment:
Seems this test is redundant with `test_files_all_file_types`. Let's remove it and add an assert for `ouvrage.name`.